cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -254,6 +254,8 @@
                             'sub_heading': sub_heading,
                             'body': parse_links(post.preview_description),
                             'links': links or null,
+                            'should_link_image': controls.should_link_image
+                                                if controls.should_link_image else false
```

Comment:
I think you can just use

```
`should_link_image': controls.should_link_image
```

because non-existence or false have the same truthiness.